src/BaseFirestoreRepository.spec.ts

```diff
@@ -362,6 +362,28 @@ describe('BaseFirestoreRepository', () => {
     });
   });
 
+  describe('findOne', () => {
+    it('must return T', async () => {
+      await bandRepository.runTransaction(async tran => {
```

Comment:
Can you remove the `runTransaction` from these tests and add another test just for the transaction?